src/Server/Utils/FileExpectationsLoader.php

```diff
@@ -52,6 +58,10 @@ public function loadExpectationFromFile($fileName)
 
         $this->logger->debug('Parsed expectation: ' . var_export($expectation, true));
         $this->storage->addExpectation($expectation);
+        // As we have no API to modify expectation parsed the same object could be used for backup.
+        // On futher changes when $expectation modifications are possible something like deep-copy
+        // should be used to clone expectation.
+        $this->backup->addExpectation($expectation);
```

Comment:
It's quicker to backup expectations than re-parse them from files.

Reply:
The amount of files will never be too high as to be a performance issue when reloading, and we avoid adding complexity and data duplication in memory.

Reply:
We would like to have all expectations defined in files to share it beetween tests for different modules in own repos, we have high count of files.